governance-exercises/VoteWizard_interpretation_exercise_01/README.md

```diff
@@ -0,0 +1,51 @@
+# VoteWizard Interpretation Exericse #1
+
+## Question #1: Do you think there is any ambiguity in these clauses? If so, where, and how would you rewrite the clauses to remove the ambiguity?
+
+### There are a few potential ambiguities
+
+- Clause 12.1 and sub-clause 12.1.2 aren't really clear about how they connect. Clause 12.1 talks about the Governance Facilitators using their judgement mainly when there are unexpected problems or mistakes in the Atlas and the Scope Artifacts. But then clause 12.1.2 says that Governance Facilitators can suggest changes whenever they want.
+
+- "WMaker" is misspelled in 12.1.1, but it's right in the original MIP113, so I'll ignore this.
+
+- Finally, 12.1.2 needs some work. It kind of gives the governance facilitator too much power to change the Scope Artifacts however they want. It doesn't really put any limits on it, and they could end up going around MIP102.
+
+### Rewritten clauses
+
+12.1: Bootstrapping Governance Votes
+Governance Facilitators are empowered to exercise broad judgment in interpreting and editing the Atlas and the Scope Artifacts. This authority should primarily be used to address unintended consequences, mistakes within these documents, or when the change cannot be implemented via the traditional governance process without leading to substantial adverse implications.
+
+12.1.1: In case of unintended consequences or mistakes in the Atlas and Scope Artifacts causing Maker Governance to function incorrectly, the Governance Facilitators can at any time run a Governance Poll to enable MKR holders to make a decision that will alleviate and resolve unintended consequences or mistakes.
+
+12.1.2: Governance Facilitators may propose edits to any content of a Scope Artifact through a Governance Poll at any time. This should only be exercised when they believe the proposed change, if implemented via the traditional MIP102 process, could lead to significant adverse consequences.
```

Comment:
Not the correct way to phrase what you want. The _change_ isn't the cause of the adverse consequences if implemented through MIP102. It's the lack of the change happening in a shorter timeframe that is the cause of adverse consequences. 

Unless I'm misunderstanding you, and you're trying to apply this to proposed-but-not-yet-included changes to the scopes which are planned to be implemented via MIP102?